### PR TITLE
elliott Bug sweep fixes

### PR DESCRIPTION
**Pull request by thegreyd**
If a tracker bug is included in assembly definition and it's build isn't found to be 
attached, it was erroneously being filtered out. Fix that.

### CODE REVIEW

File: elliott/elliottlib/bzutil.py [hunk 1]
```diff
@@ -828,7 +828,7 @@ def attach_bugs(self, bugids: List, advisory_id: int = 0, advisory_obj: Erratum
 
     def filter_bugs_by_cutoff_event(self, bugs: Iterable, desired_statuses: Iterable[str],
                                     sweep_cutoff_timestamp: float, verbose=False) -> List:
-        dt = datetime.utcfromtimestamp(sweep_cutoff_timestamp).strftime("%Y/%m/%d %H:%M")
```
Comment (thegreyd) [inline elliott/elliottlib/bzutil.py:831]: this is being deprecated

File: elliott/elliottlib/cli/find_bugs_sweep_cli.py [hunk 2]
```diff
@@ -222,6 +224,9 @@ async def find_and_attach_bugs(runtime: Runtime, advisory_id, default_advisory_t
         green_prefix(f"Searching {bug_tracker.type} for bugs with status {statuses} and target releases: {tr}\n")
 
     bugs = await get_bugs_sweep(runtime, find_bugs_obj, brew_event, bug_tracker)
+    if not bugs:
```
Comment (thegreyd) [inline elliott/elliottlib/cli/find_bugs_sweep_cli.py:227]: return early so we skip unecessary ops and logs

File: elliott/elliottlib/cli/find_bugs_sweep_cli.py [hunk 3]
```diff
@@ -376,25 +381,22 @@ def categorize_bugs_by_type(bugs: List[Bug], advisory_id_map: Dict[str, int],
                 logger.info(f"{kind} build found for #{bug.id}, {package_name} ")
                 found.add(bug)
                 bugs_by_type[kind].add(bug)
+            elif bug.id in permitted_bug_ids:
```
Comment (thegreyd) [inline elliott/elliottlib/cli/find_bugs_sweep_cli.py:384]: the actual fix

File: elliott/elliottlib/cli/find_bugs_sweep_cli.py [hunk 4]
```diff
@@ -376,25 +381,22 @@ def categorize_bugs_by_type(bugs: List[Bug], advisory_id_map: Dict[str, int],
...
+                logger.info(f'Including tracker bug #{bug.id} since it is explicitly included in the assembly config')
+                found.add(bug)
+                bugs_by_type['image'].add(bug)
```
Comment (Ximinhan) [inline elliott/elliottlib/cli/find_bugs_sweep_cli.py:387]: if this bug is related to rpm should we add it to rpm group?

Comment (vfreex) [inline elliott/elliottlib/cli/find_bugs_sweep_cli.py:387]: Even if we add the tracker to the image advisory, later `elliott attach-cve-flaws` and `elliott verify-bugs` runs will also fail due to missing build. Maybe it is more reasonable to raise an error when this occurs?

**Pull request closed without merging**